### PR TITLE
Gitlab integration doc edits

### PR DESCRIPTION
**Pull request by lauramthomas**
### Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Documentation fix (typos, incorrect content, missing content, etc.)

### Checklist

- [ ] I have read the [contributing](https://github.com/saucelabs/sauce-docs/blob/main/docs/contributing.md) document.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

**Comment by aliciakrebs** [comment]
@lauramthomas, 
Hi.

integrations-overview.md lines 16 - 18, is a card needed on this sentence?  
<div className="box boxwidetop card">
  Your CI/CD process needs automated testing in order to be successful. Sauce Labs integrates with every tool in the ecosystem.
</div>

Can you crop all of the screenshots that display a toolbar?

File: docs/integrations/tosca.md [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+
+<img src={useBaseUrl('img/integrations/tosca-test-cases-tab.png')} alt="Tosca Commander Test Cases tab"/>
+
+3. For a real device, execute the script by adding the following configuration:
+
+<img src={useBaseUrl('img/integrations/tosca-execute-real-device.png')} alt="Tosca real device configuration"/>
+
+4. For a virtual device, execute the script by adding the following configuration:
+
+<img src={useBaseUrl('img/integrations/tosca-execute-virtual-device.png')} alt="Tosca virtual device configuration"/>
```
Comment (aliciakrebs) [inline docs/integrations/tosca.md:75]:
```suggestion
<img src={useBaseUrl('img/integrations/tosca-execution-virtual-device.png')} alt="Tosca virtual device configuration"/>
```

File: docs/integrations/gitlab.md [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+
+CI/CD systems rely heavily on repeatable steps and phases. This high level of abstraction and customization makes the tool adaptable to your business needs, in that you can mix/match/repeat complex steps with a single line of config. Sauce Labs tests may be included in one or more of those steps.
+
+## .yml Files
+GitLab uses .yml for configuration. GitLab allows you to configure your environment the same way you would to run a shell script. For ultimate flexibility, GitLab allows you to launch docker images as “services”. This gives you ephemeral and/or shared APIs or other services to be available at any time in the build process, which can be created or torn down at a moment’s notice.
+
+In the example below (adapted from the [GitLab CI/CD pipeline configuration reference](https://gitlab.com/gitlab-org/gitlab/-/blob/e042b023f461be91c62d95dfd1de4547e1a8c572/doc/ci/yaml/README.md)), step 3 runs our end-to-end suite. Job names are meant to be descriptive, so we’ve named it `e2e-Sauce`.
```
Comment (aliciakrebs) [inline docs/integrations/gitlab.md:21]:
```suggestion
In the example below (adapted from the [GitLab CI/CD pipeline configuration reference](https://gitlab.com/gitlab-org/gitlab/-/blob/e042b023f461be91c62d95dfd1de4547e1a8c572/doc/ci/yaml/README.md)), job 3 runs our end-to-end suite. Job names are meant to be descriptive, so we’ve named it `e2e-Sauce`.
```

**Review by mmerrell — APPROVED** [review]
This all looks good to me--can't wait to see it go live!

**Pull request closed without merging**